src/ucp/api/ucp.h

```diff
@@ -532,19 +546,32 @@ enum ucp_datatype_attr_field {
  * ucp_mem_map() function.
  */
 enum {
-    UCP_MEM_MAP_NONBLOCK = UCS_BIT(0), /**< Complete the mapping faster, possibly by
-                                            not populating the pages in the mapping
-                                            up-front, and mapping them later when
-                                            they are accessed by communication
-                                            routines. */
-    UCP_MEM_MAP_ALLOCATE = UCS_BIT(1), /**< Identify requirement for allocation,
-                                            if passed address is not a null-pointer
-                                            then it will be used as a hint or direct
-                                            address for allocation. */
-    UCP_MEM_MAP_FIXED    = UCS_BIT(2)  /**< Don't interpret address as a hint:
-                                            place the mapping at exactly that
-                                            address. The address must be a multiple
-                                            of the page size. */
+    /**
+     * Complete the mapping faster, possibly by not populating the pages in the
+     * mapping up-front, and mapping them later when they are accessed by
+     * communication routines.
+     */
+    UCP_MEM_MAP_NONBLOCK = UCS_BIT(0),
+
+    /**
+     * Identify requirement for allocation, if passed address is not a
+     * null-pointer then it will be used as a hint or direct address for
+     * allocation.
```

Comment:
Think we can simplify here:

If passed address is not a
null-pointer, then it will be used as a hint or direct address for
allocation.

Reply:
done